packages/gen-ai-hub/README.md

```diff
@@ -0,0 +1,32 @@
+# @sap-ai-sdk/gen-ai-hub
+
+This package incorporates generative AI into your AI activities in SAP AI Core and SAP AI Launchpad.
+
+### Installation
+
+```
+$ npm install @sap-ai-sdk/gen-ai-hub
+```
+
+## Pre-requisites
+
+- [Enable the AI Core service in BTP](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/initial-setup)
+- Project configured with Node.js v20 or higher and native ESM support enabled
+- For testing your application locally:
+  - Download a service key for your AI Core service instance
+  - Create a `.env` file in the sample-code directory
+  - Add an entry `AICORE_SERVICE_KEY='<content-of-service-key>'`
+
+## Usage
+
+<!-- Details to be added -->
+
+## Support, Feedback, Contribution
+
+This project is open to feature requests/suggestions, bug reports etc. via [GitHub issues](https://github.com/SAP/ai-sdk-js/issues).
+
+Contribution and feedback are encouraged and always welcome. For more information about how to contribute, the project structure, as well as additional contribution information, see our [Contribution Guidelines](https://github.com/SAP/ai-sdk-js/blob/main/CONTRIBUTING.md).
+
+## License
+
+The SAP AI SDK is released under the [Apache License Version 2.0.](http://www.apache.org/licenses/)
```

Comment:
[q] Same as above, can we add it only in the root?

Reply:
The README contents are displayed on npmjs. So this info needs to be there for every package.